ext/pdo/tests/pdo_test.inc

```diff
@@ -82,5 +69,27 @@ class PDOTest {
 
         return $config;
     }
+
+    public static function dropTableIfExists(PDO $db, string $tableName): void {
+        if (getenv('PDOTEST_DSN') === "sqlite::memory:") {
+            return;
+        }
+
+        match ($db->getAttribute(PDO::ATTR_DRIVER_NAME)) {
+            "oci" => $db->exec(<<<SQL
+                 BEGIN
+                     EXECUTE IMMEDIATE 'DROP TABLE $tableName';
+                 EXCEPTION
+                     WHEN OTHERS THEN
+                         IF SQLCODE != -942 THEN
+                             RAISE;
+                         END IF;
+                    END;
+                SQL
+            ),
+            "firebird" => $db->exec("DROP TABLE $tableName"),
```

Comment:
Seems like ``EXISTS`` is support but in a convoluted way https://firebirdsql.org/file/documentation/chunk/en/refdocs/fblangref40/fblangref40-commons-predicates.html#fblangref40-commons-existential

Reply:
Ah, no way... :/ However, I noticed that some existing firebird tests suppresses the warnings (via `@`) during the `DROP TABLE`s, so I think that's what I should continue to do.